README.md

```diff
@@ -58,7 +60,119 @@ Master and Worker nodes:
 2. Create a stack in Heat by passing the template, environment file, and parameters:
 ```
   $ openstack stack create -t caasp-stack.yaml -e caasp-environment.yaml \
-    --parameter image=CaaSP-2 caasp-stack
+    --parameter image=CaaSP-3 caasp-stack
+```
+
+### INSTALL TEMPLATES FROM COMMAND LINE (FOR MULTIPLE MASTER NODES)
```

Comment:
Can we add a note that this cannot be done via horizon.

Reply:
done